addons/mail/views/mail_message_reaction_views.xml

```diff
@@ -1,12 +1,33 @@
 <?xml version="1.0"?>
 <odoo>
 
+    <record id="mail_message_reaction_view_form" model="ir.ui.view">
+        <field name="name">mail.message.reaction.form</field>
+        <field name="model">mail.message.reaction</field>
+        <field name="arch" type="xml">
+            <form string="Reactions" edit="0">
+                <sheet>
+                    <group>
+                        <group>
+                            <field name="message_id"/>
+                            <field name="content"/>
+                        </group>
+                        <group>
+                            <field name="partner_id"/>
+                            <field name="guest_id"/>
+                        </group>
+                    </group>
+                </sheet>
+            </form>
+        </field>
+    </record>
+
     <record id="mail_message_reaction_view_tree" model="ir.ui.view">
         <field name="name">mail.message.reaction.tree</field>
         <field name="model">mail.message.reaction</field>
         <field name="priority">10</field>
         <field name="arch" type="xml">
-            <tree string="Reactions">
+            <tree string="Reactions" create="0">
```

Comment:
Isn't there also a way to prevent create from the form view? Because it doesn't work from there either.

Reply:
removed create option from form view too